README.rst

```diff
@@ -18,6 +18,21 @@ Python Client for Analytics Admin API
 .. _Client Library Documentation: https://googleapis.dev/python/analyticsadmin/latest
 .. _Product Documentation:  https://developers.google.com/analytics/
 
+Quick Start
+-----------
+
+In order to use this library, you first need to go through the following steps:
+
+1. `Select or create a Cloud Platform project.`_
```

Comment:
Analytics is not a part of cloud. Is the "create a project" flow the same?

Reply:
Great catch! Fixed.